package.json

```diff
@@ -6,7 +6,9 @@
   "scripts": {
     "test": "echo \"Error: no test specified\" && exit 1",
     "lint": "sass-lint 'src/**/*.scss' -v",
-    "prepublish": "npm run lint"
+    "prepublish": "npm run lint",
+    "install": "gulp build",
```

Comment:
I think using `postinstall` is a better method for compiling locally after install. My only worry with overloading install is it could produce unintended side effects when running this through various CI platforms?

Reply:
I'm fine with `postinstall`. Not sure the implication of CI. If we run into issues it could be removed from the task altogether.

Reply:
It looks like `install` and `postinstall` both run after the package is installed: https://docs.npmjs.com/misc/scripts